image_proc/CMakeLists.txt

```diff
@@ -14,213 +14,92 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_components REQUIRED)
-find_package(rcutils REQUIRED)
 find_package(cv_bridge REQUIRED)
 find_package(image_transport REQUIRED)
 find_package(sensor_msgs REQUIRED)
-find_package(OpenCV 3 REQUIRED)
+find_package(OpenCV 4 REQUIRED)
```

Comment:
Dashing and Eloquent's target is still openCV3, but you're right that I believe Foxy will be opencv 4. We should deal with branching at then at least @JWhitleyWork